app/styles/details.scss

```diff
@@ -5,7 +5,7 @@
   .card {
     box-shadow: 2px 2px 3px black;
     padding: 15px;
-    width: 100%;
+    width: 100%!important;
```

Comment:
Try avoid use `!importat` - in 99.9% you cannot use it 😃 through using correct specificity - cheatsheet: https://specifishity.com/